code/modules/planar_lighting/light_effect.dm

```diff
@@ -13,18 +13,38 @@ var/list/light_over_cache = list()
 	glide_size = 32
 	blend_mode = BLEND_ADD
 
-	var/image/light_overlay
+	light_range = 5
+
+//	var/image/light_overlay
 	var/current_power = 1
 	var/atom/movable/holder
 	var/point_angle
 	var/list/affecting_turfs = list()
 
 /obj/light/New(var/newholder)
 	holder = newholder
-	light_overlay = image(icon = 'icons/planar_lighting/lighting_overlays.dmi', icon_state = holder.light_type)
+
+	light_range = max(0, min(5, light_range))
+	pixel_x = pixel_y = -(world.icon_size * light_range)
```

Comment:
`x = y = z` is only supported in 511+; this will need to be `x = z; y = x` or similar to run on 510.